### ISSUE TITLE
[BUG] Using keyboard shortcut j makes feed jump

### ISSUE THREAD

**Issue by poblabs** [FreshRSS/FreshRSS#5551]
**Describe the bug**
I am new to FreshRSS so I am checking it out. When I am browsing the main stream and use hotkey `j` to go to the next article, the whole stream jumps up and goes out of view. I then have to use my mouse to bring it back to view - which defeats the purpose of the hotkeys. 

**To Reproduce**
1. Go to Main Stream
2. Press j
3. Wait a second
4. Stream jumps

**Expected behavior**
Stream to not jump and stay on the current article - top aligned.

**Screenshots**
Start of the main stream:

![image](https://github.com/FreshRSS/FreshRSS/assets/3484775/c34cc426-9ee1-41f6-b058-2cd14c455c09)

After pressing `j`, it goes to the next article but then it jumps and results in not being top-aligned. You can see how this comic is cut off at the top. 

![image](https://github.com/FreshRSS/FreshRSS/assets/3484775/1242a287-389f-4a6c-9728-b497148b4bdb)

The full article should have looked like this (not cut off after a weird jump):

![image](https://github.com/FreshRSS/FreshRSS/assets/3484775/dcbc048d-df18-4a1a-9901-0f907727ba60)

**Environment information (please complete the following information):**
- Device: Desktop Computer
- OS: Windows 11
- Browser: Chrome Version 115.0.5790.110 (Official Build) (64-bit)
- FreshRSS version: 1.21.0
- Installation type: linuxserver Docker image

**Comment by Frenzie** [comment]
Can't confirm (whether on that exact same version of Chrome except on Linux or in Firefox 115.0.2). But that was to be expected; I'd have noticed long ago if I could.

Do you have any extensions that could be involved?

**Comment by Alkarex** [comment]
Try also the following option:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/58091ce7-e9d7-410f-b749-31dd0d270e95)

**Comment by poblabs** [comment]
I downloaded firefox with no extensions to give this a try and the jumping behavior is happening again. 

I've disabled lazy load of images, and have that option selected @Alkarex . Not sure what else to look for. 

One thing I've noticed is that it seems to happen if I press `j` faster than the article is marked as read. Maybe that's something? Is there a way to mark the article read instantly?

There are some feeds I have from twitter that are mostly retweets or something, so I fly through those pretty fast. 

![image](https://github.com/FreshRSS/FreshRSS/assets/3484775/23cd5f26-d9d4-4943-98a7-b8e89030e6d3)

**Comment by Frenzie** [comment]
Pardon, by extensions I meant FreshRSS extensions, though nothing wrong with ruling out browser extensions too.

> One thing I've noticed is that it seems to happen if I press j faster than the article is marked as read. Maybe that's something? Is there a way to mark the article read instantly?

That's normal and done separately on the side. That isn't to say there couldn't be something to what you're saying about doing it very quickly but it shouldn't be related to the articles being marked as read. (That being said I regularly press J and K very quickly.)

**Comment by poblabs** [comment]
Got it, and nope - no FreshRSS extensions. I did a little screen recording to try and show what I mean. It didn't capture my key presses, but there were about 1 second apart. You can see the articles start jumping around. 

Not sure what else to do

https://github.com/FreshRSS/FreshRSS/assets/3484775/d1a909f9-105e-48a0-9046-c56e91f5a904

**Comment by poblabs** [comment]
Update, found this utility that can show my keyboard presses on screen. Hope this helps?


https://github.com/FreshRSS/FreshRSS/assets/3484775/751b8df5-28e6-48dc-95bc-958c917673aa

**Comment by Frenzie** [comment]
Yeah, keypress-wise I'm not seeing anything I don't do. It's possible I have fewer images in my feeds which potentially comes back to the lazyloading but I also have plenty that do.

Maybe something less typical like embeds?

**Comment by Alkarex** [comment]
Can you remember having changed any settings, for instance *Number of articles per page*:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/95630542-82a7-4b94-ac31-82cff91184f2)

In any case, can you try with a higher value such as 30?

**Comment by poblabs** [comment]
> Can you remember having changed any settings, for instance _Number of articles per page_:
> 
> 
> In any case, can you try with a higher value such as 30?

No change in behavior. Setting to 30, and unchecking "Load more articles at the bottom of the page" still result in the article jumping
